content/codespaces/prebuilding-your-codespaces/managing-prebuilds.md

```diff
@@ -52,13 +52,13 @@ This displays the workflow run history for prebuilds for the associated branch.
 ## Editing a prebuild configuration
 
 1. On the {% data variables.product.prodname_codespaces %} page of your repository settings, click the ellipsis to the right of the prebuild configuration you want to edit.
-1. In the dropdown menu, click **Edit**.
+2. In the dropdown menu, click **Edit**.
 
-   ![Screenshot of the options dropdown menu for a configuration, displayed by clicking a button labeled with three dots. The "Edit" option is selected.](/assets/images/help/codespaces/prebuilds-edit.png)
+![Screenshot of the options dropdown menu for a configuration, displayed by clicking a button labeled with three dots. The "Edit" option is selected.](/assets/images/help/codespaces/prebuilds-edit.png)
```

Comment:
Screenshots within procedural steps must be intended to be part of the procedural step. Otherwise, they will cause the numbering to restart at 1.

```suggestion
   ![Screenshot of the options dropdown menu for a configuration, displayed by clicking a button labeled with three dots. The "Edit" option is selected.](/assets/images/help/codespaces/prebuilds-edit.png)
```